### PR TITLE
Display poi email under website block

### PR DESCRIPTION
**Pull request by Julien-laville**
mail

### CODE REVIEW

File: src/scss/includes/font.scss [hunk 1]
```diff
@@ -28,14 +28,15 @@
 
 .icon-pin_geoloc:before {
   content: "\e90d";
+  color: #5c6f84;
```
Comment (amatissart) [inline src/scss/includes/font.scss]: why this change ?

File: src/scss/includes/font.scss [hunk 2]
```diff
@@ -28,14 +28,15 @@
...
 }
 .icon-check:before {
   content: "\e91c";
 }
 .icon-chevron-left:before {
   content: "\e913";
 }
-.icon-chevron-right:before {
+.icon-chevron-right2:before {
```
Comment (amatissart) [inline src/scss/includes/font.scss]: why this change ?

File: src/scss/includes/font.scss (hunk 1 above)
Comment (Julien-laville) [reply src/scss/includes/font.scss]: argghhhhh ico moon corrupted

File: src/scss/includes/font.scss (hunk 2 above)
Comment (Julien-laville) [reply src/scss/includes/font.scss]: when i update icomoon i need to remove the 2 for left, i missed this time

**Comment by amatissart** [comment]
Please add a test case :)
